src/services/user.service.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-return-await */
+
+const { User } = require('../models/User.model');
+
+const getAll = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getById = async (id) => {
+  return await User.findByPk(id);
+};
+
+const createUser = async (name) => {
+  return User.create({ name });
```

Comment:
Consider adding validation for the `name` parameter in the `createUser` function to ensure it is a valid string before creating a new user.